app/views/account/entries/entryables/transaction/_transaction.html.erb

```diff
@@ -51,6 +52,12 @@
     <% end %>
   </div>
 
+  <% if is_investment_transfer %>
+    <div class="col-span-5 text-right">
+      <%= tag.p entry.inflow? ? t(".deposit") : t(".withdrawal") %>
+    </div>
+  <% end %>
```

Comment:
This partial is starting to mix a lot of logic related to the display of transfers.   It will likely need a refactor soon for readability and future flexibility, but leaving it as-is until we've nailed down all of the account views so all requirements are known prior to reworking.